concordium-consensus/src/Concordium/GlobalState/BlockState.hs

```diff
@@ -1521,6 +1521,10 @@ class (BlockStateQuery m, PLTQuery (UpdatableBlockState m) m) => BlockStateOpera
         (UpdateValue (ChainParametersVersionFor (MPV m))) ->
         m (UpdatableBlockState m)
 
+    -- | Increment the update sequence number for Protocol Level Tokens (PLT).
+    -- Unlike the other chain updates this is a separate function, since there is no queue associated with PLTs.
+    bsoIncrementPltUpdateSequenceNumber :: UpdatableBlockState m -> m (UpdatableBlockState m)
```

Comment:
Minor gripe: I'd prefer the style `bsoIncrementPLTUpdateSequenceNumber`, which is more consistent with what we use elsewhere.

Reply:
I'll fix